micrometer-core/src/main/java/io/micrometer/core/instrument/binder/jvm/JvmThreadMetrics.java

```diff
@@ -15,6 +15,12 @@
  */
 package io.micrometer.core.instrument.binder.jvm;
 
+import io.micrometer.common.lang.NonNullApi;
```

Comment:
I believe from your earlier PR `java.*` imports go first and then `io.*` imports. see [this commit](https://github.com/micrometer-metrics/micrometer/pull/3651/commits/4bb8b69ad32d6c68c5d8a7bfe564395a20ac6561) by @jonatan-ivanov

Reply:
Here I compared the previous code, I did not find anything wrong, is that the order of import?